Multisig/UI/App/MainTabView.swift

```diff
@@ -59,11 +58,10 @@ struct MainTabView: View {
         }
         .onAppear {
             // this adjusts the snack bar position
-            App.shared.snackbar.bottomEdgeSpacing =
-                BottomTabViewMetrics.tabBarHeight + self.tabBarSpacing
+            App.shared.snackbar.pushBottomPadding(ScreenMetrics.aboveTabBar)
         }
         .onDisappear {
-            App.shared.snackbar.bottomEdgeSpacing = 0
+            App.shared.snackbar.popBottomPadding()
```

Comment:
I think Set/Reset is better that push/pop

Reply:
ok